docs/cow-protocol/tutorials/widget/widget.md

```diff
@@ -131,24 +131,28 @@ createCowSwapWidget(
+| `images`               | `CowSwapWidgetImages`  | ---                    | To make your widget unique, you can replace some of the images. For example, a stub image when there are no limit orders.                                                                                                                                                                                                   |
+| `banners`              | `CowSwapWidgetBanners` | ---                    | With this option you can control the display of some banners in the CoW Swap interface.                                                                                                                                                                                                                                     |
+| `tokenLists`           | `string[]`             | ---                    | CowSwap uses the [token lists standard](https://tokenlists.org), which allows flexible and decentralized management of assets for trading.                                                                                                                                                                                  |
+| `customTokens`         | `TokenInfo[]`          | ---                    | In the case when you want to use tokens that are not included in the default token lists, you can add them programmatically by simply passing an array of ERC-20 tokens info.                                                                                                                                               |
+| `disableToastMessages` | `boolean`              | false                  | CoW Swap displays a pop-up notification when certain events occur, for example: an order has been filled. You may want to handle these events yourself and disable the display of notifications in the widget, to do this you need to enable this option. See [Events handling](#events-handling) section for more details. |
+| `standaloneMode`       | `boolean`              | false                  | When this option is enabled, the widget will use its own Ethereum provider and the user will be able to connect a wallet from within the widget.                                                                                                                                                                            |
+| `hideLogo`             | `boolean`              | false                  | Option to hide the logo in the widget.                                                                                                                                                                                                                                                                                      |
```

Comment:
do we still use this setting? do we need it?

Reply:
Nice catch!
Removed it